src/app/api/profiles/[id]/reject/route.ts

```diff
@@ -1,30 +1,44 @@
-import { saveRejectingReason } from '@/backend/profile/rejection.service'
+import { sendProfileRejectedEmail } from '@/actions/mailing/sendProfileStatusEmail'
```

Comment:
**nit:** we should not import from `actions` to `service`. Only from `service` to `actions`. If you are missing service method just create `backend/mailing/mailing.service.ts` where you will keep reusable mailing pieces. Then you can import it here or to `actions`